source/base/mpi.cc

```diff
@@ -68,6 +68,24 @@ DEAL_II_NAMESPACE_OPEN
 
 namespace Utilities
 {
+  IndexSet
+  create_evenly_distributed_partitioning(const unsigned int &my_partition_id,
```

Comment:
Let's not pass `unsigned int` as reference.
```suggestion
  create_evenly_distributed_partitioning(const unsigned int my_partition_id,
```